tools/pontoon/search_filters.md

```diff
@@ -0,0 +1,30 @@
+# Search and filters
+
+A string in Pontoon can be in one of these states:
+* *Missing*: string is not available in the localized file.
+* *Suggested*: at least one translation has been suggested for this string. Note that, in this case, the translation only exists within Pontoon database, it’s not exported to external repositories.
+* *Translated*: string has an approved translation.
+* *Fuzzy*: string is marked as fuzzy in the localized file. This state is possible only if the source is a [Gettext](https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html) (.po) file. Translation is in the file, marked as fuzzy and therefore not used by the website or application.
```

Comment:
For the new contributor, it may not be clear what "fuzzy" means here. It is clear that the translation exists in Pontoon and not in the website or app, but not what "fuzzy" means. 

Are we concerned with making this explicit in this doc?